src/Utilities/model.jl

```diff
@@ -318,6 +327,62 @@ function MOI.get(model::AbstractModel, ::MOI.ListOfModelAttributesSet)::Vector{M
 end
 
 # Constraints
+single_variable_flag(::Type{<:MOI.EqualTo}) = 0x1
+single_variable_flag(::Type{<:MOI.GreaterThan}) = 0x2
+single_variable_flag(::Type{<:MOI.LessThan}) = 0x4
+single_variable_flag(::Type{<:MOI.Interval}) = 0x8
+single_variable_flag(::Type{MOI.Integer}) = 0x10
+single_variable_flag(::Type{MOI.ZeroOne}) = 0x20
+single_variable_flag(::Type{MOI.Semicontinuous}) = 0x40
+single_variable_flag(::Type{MOI.Semiinteger}) = 0x80
+
+# Sets setting lower bound:
```

Comment:
A "lower bound" of a set doesn't make much sense in isolation, consider another name. Maybe `extract_lower_bound`?